### PR TITLE
[hotfix][3.6.3] Merging back hotfix into finney

### PR DESCRIPTION
**Pull request by eduardogr**

### CODE REVIEW

File: bittensor/_cli/cli_impl.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def __init__(self, config: 'bittensor.Config' ):
                 config (:obj:`bittensor.Config`, `required`): 
                     bittensor.cli.config()
         """
-        if not config.no_version_checking:
+        if config.get('no_version_checking') != None and not config.no_version_checking:
```
Comment (camfairchild) [inline bittensor/_cli/cli_impl.py]: Could be `if not config.get('no_version_checking'):`   

because None is falsey and indicates False here also

**Review by camfairchild — COMMENTED** [review]
LGTM. Just a nit

File: bittensor/_cli/cli_impl.py (hunk 1 above)
Comment (eduardogr) [reply bittensor/_cli/cli_impl.py]: please, add the code suggestion and we add it right away :)

**Review by camfairchild — APPROVED** [review]
lgtm

**Review by Eugene-hu — APPROVED** [review]

**Pull request merged**